missions/g4/transcripts/TEC

```diff
@@ -8047,7 +8047,7 @@ CC: Gemini 4, this is Houston CAP COM. I'm not receiving
      you. I'll broadcast the last two items of the Flight
      Plan in the blind. Command Pilot eat 07:15 to 11:15.
      Pilot sleeps 07:15 to 11:15. This is Houston CAP
-     COM broadcasting in the blind, Flight Plan 08:35
+     COM broadcasting in the blind, Flight Plan 08:35 RKV .....
```

Comment:
We usually trim ellipses down three dots, because there's some code in the Web site that converts `...` to the `…` character.